### DISCUSSION TITLE
[RFC Call] Agenda - February 22, 2021

### DISCUSSION THREAD

**Discussion by FredKSchott** [withastro/roadmap#125] [Announcements]
# To Discuss

- @Siilwyn https://github.com/withastro/rfcs/pull/121

_If there is something that you would like to see discussed, please comment below. We will prioritize pending RFC, but if we have time to spare we will open the call up to get attention on and discuss [other threads and feature requests.](https://github.com/withastro/rfcs/discussions)._

**Comment by boehs** [comment]
These should be non controversial and can likely be 5-10 minutes a pop. I basically want to get approval and then merge full RfC and/or implement

# Render directive

https://github.com/withastro/rfcs/discussions/126

# const defer

https://github.com/withastro/rfcs/discussions/127

**Comment by Tc-001** [comment]
We could try and see if https://github.com/withastro/rfcs/discussions/116 (SSR islands) is possible

**Comment by tony-sull** [comment]
[#104](https://github.com/withastro/rfcs/discussions/104) is also very related to the topics above (async styles, more control over CSS chunking/bundling, etc)
